frontend/.env.sample

```diff
@@ -1,3 +1,4 @@
 API_ENDPOINT = http://localhost:3000
 RAZORPAY_KEY_ID = <YOUR_RAZOR_PAY_KEY_ID>
-CLIENT_URL = <frontend_deployed_url>
\ No newline at end of file
+CLIENT_URL = <frontend_deployed_url>
+GOOGLE_API_KEY = <YOUR_GOOGLE_API_KEY> // make sure to enable places api
```

Comment:
is this the android Auth Id?

Reply:
https://www.npmjs.com/package/react-native-google-places-autocomplete



Follow these steps to get the api and enable places api. This npm is not working as expected so created a component.